docker/volumes/db/init/migration.sql

```diff
@@ -1,3 +1,23 @@
+-- delete entries that share the same user id, isrc, and listen timestamp
+delete from prod.played_tracks
+where play_id in (
+  select T1.play_id from prod.played_tracks T1, prod.played_tracks T2
+  where T1.user_id = T2.user_id
+  and T1.isrc = T2.isrc
+  and T1.listened_at = T2.listened_at
+  and T1.play_id < T2.play_id
+);
+
+delete from test.played_tracks
+where play_id in (
+  select T1.play_id from test.played_tracks T1, test.played_tracks T2
+  where T1.user_id = T2.user_id
+  and T1.isrc = T2.isrc
+  and T1.listened_at = T2.listened_at
+  and T1.play_id < T2.play_id
+);
+
+-- change date structure for album release dates
```

Comment:
erm i think you mean structured data

![image](https://github.com/user-attachments/assets/6bd6f6a1-23a8-4c0b-b9de-f089c7ab8e4a)